FlySimEmbedded/FlySim.ino

```diff
@@ -159,15 +159,16 @@ void loop()
             char messagePayload[MESSAGE_MAX_LEN];
 
             bool temperatureAlert = readMessage(messageCount++, messagePayload, MESSAGE_MAX_LEN);
+            LogInfo("Info: Sending flight data: %s", messagePayload);
```

Comment:
Print out too many messages is not good and bothering people, consider to remove this line

Reply:
This is to be consistent with the UI. The origin image and output of the code are different.